csp/gitprojectsettings.csp

```diff
@@ -564,6 +576,26 @@ body {
             </div> -->
         </div>
 
+        <div class="form-group row mb-3">
+            <label for="addToFav" class="offset-sm-1 col-sm-3 col-form-label"  data-toggle="tooltip" data-placement="top" title="Favorite namespaces for user. For Windows/Mac: Hold down the control(ctrl)/command(cmd) button to select multiple options.">Favorite Namespaces</label>
```

Comment:
The instruction to use should be visible without hover (maybe below / above the box), since usage is not fully intuitive